src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -91,6 +91,7 @@ class HoldRequest extends React.Component {
     let path = `${appConfig.baseUrl}/hold/confirmation/${bibId}-${itemId}`;
     const searchKeywordsQuery =
       (this.props.searchKeywords) ? `searchKeywords=${this.props.searchKeywords}` : '';
+    const searchKeywordsQueryPhysical = searchKeywordsQuery ? `&${searchKeywordsQuery}` : '';
 
     if (this.state.delivery === 'edd') {
       const searchKeywordsQueryEdd = searchKeywordsQuery ? `?${searchKeywordsQuery}` : '';
```

Comment:
I understand what you are pointing here! I don't like an extra `?` or `&` in the end of the URL though. How do you think? Is this worthy to use two variables? Or you think the extra character is fine?